test/msw-api/setup-server/scenarios/on-unhandled-request/warn.test.ts

```diff
@@ -18,12 +18,18 @@ afterAll(() => {
 })
 
 test('warns on unhandled request when using the "warn" value', async () => {
-  jest.spyOn(global.console, 'warn')
+  const consoleSpy = jest.spyOn(global.console, 'warn')
```

Comment:
Please remove the spy by calling `jest.clearAllMocks()`. `afterEach` may be a good place to do that.